influxql/functions.go

```diff
@@ -211,29 +211,31 @@ func MapMean(itr Iterator) interface{} {
 
 	for _, k, v := itr.Next(); k != 0; _, k, v = itr.Next() {
 		out.Count++
-		out.Sum += v.(float64)
+		out.Mean += (v.(float64) - out.Mean) / float64(out.Count)
 	}
 	return out
 }
 
 type meanMapOutput struct {
 	Count int
```

Comment:
I think leaving count as an integer makes sense to me. I don't think the cast is particularly expensive, so I'd leave it that way for now unless you feel strongly otherwise.